cypress/plugins/index.js

```diff
@@ -16,19 +16,34 @@ module.exports = (on, config) => {
   // `on` is used to hook into various events Cypress emits
   // `config` is the resolved Cypress config
 
-  // on('before:browser:launch', (browser = {}, args) => {
-  //   if (browser.name === 'chrome') {
-  //     args.push('--disable-blink-features=RootLayerScrolling')
-  //     return args
-  //   }
-  // })
+  on('before:browser:launch', (browser = {}, args) => {
+    //require('../GenerateCypressTestSpecs')
+    
+    // BUG BUG - Need to work out why this code does not execute or where it is throwing an exception.
+    alert('got here');
+    var fs = require('../../node_modules/pn/fs')
+    var logFile = fs.openSync('c:/temp/GenerateCypressTestSpecs.log', 'a');
+    fs.appendFileSync(logFile, message);
+    fs.closeSync(logFile);
+
+    // if (browser.name === 'chrome') {
+    //   args.push('--disable-blink-features=RootLayerScrolling')
+    //   return args
+    // }
+  })
+
   on('task', {
     log: (message) => {
       console.log(message)
       return null
     }, 
     parse: (filePath) => {
       return path.parse(path.normalize(filePath))
-    }
+    },
+    exists: (path) => {
+      return new Promise(resolve => { 
+        fs.access(path, fs.constants.F_OK, (err) => { resolve(err ? false : true) }) 
```

Comment:
It's weird to use `fs` variable when it's not declared in scope. The seems to only leverage `var` being global and assigned by the time this is called.  I would think it could be `undefined`

Reply:
Fixed